src/views/ModelPlan/Table/index.tsx

```diff
@@ -195,7 +195,7 @@ const DraftModelPlansTable = ({
       },
       {
         Header: t('requestsTable.headers.recentActivity'),
-        accessor: 'createdDts',
+        accessor: 'modifiedDts',
         Cell: ({
           row,
           value
```

Comment:
Getting this situation when I first seed the models
![image](https://user-images.githubusercontent.com/2349702/217887342-1e460993-9438-482d-8d7f-ae50d0a581aa.png)

Reply:
Oh snap, how did I miss that, thanks